src/components/header.js

```diff
@@ -99,11 +108,13 @@ const Header = () => {
 
   return (
     <HeaderUSWDS extended={true}>
-      {/* <a class="usa-skipnav" href="#main-content">Skip to main content</a> */}
+      <a class="usa-skipnav" href="#main-content">
```

Comment:
I think there might be a way to do this through gatsby's Link component

Reply:
They have something even better with `import { SkipNavLink, SkipNavContent } from "@reach/skip-nav";`. Check out the new push.